src/models/Settings.js

```diff
@@ -314,6 +314,9 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
     },
 
     setAuthClient: function (authClient) {
+      if (authClient && authClient.options && authClient.options.headers) {
+        authClient.options.headers['Accept-Language'] = this.get('languageCode');
```

Comment:
I think `languageCode` always returns a value right? (default 'en') So if the user does not set a language, then `get('languageCode')` will return English by default. This could be a problem if the browser is in a different language than English, cause the widget will get the browser language, but the API will be called with 'en'

Reply:
Hey @mauriciocastillosilva-okta if the locale is not set via widget config but instead we use browser setting to change the locale to something other than english,  `languageCode ` will get that value set in the browser's language setting. Internally while calculating `languageCode ` we call `BrowserFeatures.getUserLanguages()` https://github.com/okta/okta-signin-widget/blob/master/src/util/BrowserFeatures.js#L58 

Screenshot below shows widget not configured with any locale but browser locale set to spanish, and `languageCode` picking up the browser setting

<img width="1425" alt="screen shot 2018-06-08 at 7 05 38 am" src="https://user-images.githubusercontent.com/23267876/41168339-7dffc964-6afa-11e8-84e0-5ed212b40f02.png">

Reply:
@santhoshbalakrishnan this is what we were wondering. As @nikhilvenkatraman-okta pointed out, languageCode considers the browser languages in its logic, so if the config language is invalid, we check then if the browser languages are supported. And if none are, then we default to en.
@nikhilvenkatraman-okta can you double check we have a test for languageCode where the config lang is invalid so we return the browser lang?

Reply:
@mauriciocastillosilva-okta https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/LoginRouter_spec.js#L1378